### PR TITLE
Fix testing rules and runner to work under Windows

### PR DESCRIPTION
**Pull request by vmax**
## What is the goal of this PR?

Fix the testing infrastructure to allow testing Grakn Core distribution under Windows.

## What are the changes implemented in this PR?

* `grakn_java_test` rule macro no longer accepts `select` and instead takes in the native artifact mapping in this form:
```
grakn_java_test(
    mac_artifact = "//server:assemble-mac-zip",
    linux_artifact =  "//server:assemble-linux-targz",
    windows_artifact = "//server:assemble-windows-zip",
}
```

This allows to pass the correct platform-specific artifact and preserve its extension to the test (previously they would all end up under fixed name of `grakn-core-native-artifact.tar.gz`, for Linux, Windows and Mac artifacts making it impossible for `GraknCoreRunner` to tell for which distribution is the artifact for)

More about this implementation of this approach [here](https://docs.bazel.build/versions/master/configurable-attributes.html#can-i-read-select-like-a-dict).

* Binary scripts (`grakn` and `grakn.bat`) now do a simpler check on whether the Console or Server are included in the distribution being ran. This allows the check that previously was false-negative due to long filenames to pass on Windows.

* `GraknCoreRunner` now distinguishes platforms and picks the correct binary script (`grakn` vs `grakn.bat`). Additionally, test timeout is increased.

### CODE REVIEW

**Comment by haikalpribadi** [comment]
Can I leave this to @lolski to review?

**Comment by vmax** [comment]
@haikalpribadi yeah it's just some build stuff, we'll sort it out with Ganesh

File: binary/grakn.bat [hunk 1]
```diff
@@ -54,7 +54,7 @@ goto exiterror
 :startconsole
 
 set "G_CP=%GRAKN_HOME%\console\conf\;%GRAKN_HOME%\console\services\lib\*"
-if exist .\console\services\lib\io-grakn-console-grakn-console-*.jar (
```
Comment (lolski) [inline binary/grakn.bat:57]: can we also do the same change in `binary/grakn` script? i think this is much cleaner

Comment (vmax) [reply binary/grakn.bat:57]: @lolski addressed in https://github.com/graknlabs/common/pull/81/commits/19e19e0f8eb8695166a6dbb0c3a1cc1d2e463870

**Review by lolski — APPROVED** [review]

**Pull request merged**